public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRuleModal.test.tsx

```diff
@@ -18,31 +17,97 @@ jest.mock('app/core/app_events', () => {
 });
 
 describe('AddAlertRuleModal', () => {
+  const initialValues: AlertRule = {
+    ruleId: '/rule_id/ded33d30-1b65-4b43-ba45-75ca52b48fa5',
+    createdAt: '2021-01-19 12:53:16.082',
+    duration: '3600s',
+    filters: [],
+    severity: 'Critical',
+    summary: 'Just a summary',
+    threshold: '1 %',
+    lastNotified: '',
+    disabled: false,
+    rawValues: {
+      channels: [],
+      filters: [],
+      disabled: false,
+      template: {
+        name: 'pmm_mongodb_connections_memory_usage',
+        summary: 'Memory used by MongoDB connections',
+        params: [],
+      },
+      rule_id: '/rule_id/ded33d30-1b65-4b43-ba45-75ca52b48fa5',
+      summary: 'Just a summary',
+      params: [],
+      for: '3600s',
+      severity: 'SEVERITY_CRITICAL',
+      created_at: '2021-01-19T12:53:16.082610Z',
+    },
+  };
+
   afterEach(() => {
     jest.clearAllMocks();
   });
 
-  it('doesn not render the modal when visible is set to false', async () => {
-    let wrapper: ReactWrapper<{}, {}, any>;
+  it('should correctly render modal', () => {
```

Comment:
nit: `Should render the modal`, since we the reader wouldn't know what `correctly` means, nor do we want them to guess. The goal of a test is that the component does it the way we programmed it to, so `correctly` is always meant.

Reply:
Definitely ;)